workflows/epigenetics/CUTandRUN/CUTandRUN-tests.yml

```
@@ -0,0 +1,69 @@
+- doc: Test outline for CUTandRUN
```

This is the yaml list syntax, you can add another test like this:
```suggestion
- doc: Test outline for CUTandRUN (dm6)
   job:
     PE fastq input:
     ...
- doc: Test outline for CUTandRUN (hg38)
```

---

thanks